feature/lines+header+mobile menu

HaidaienkoK opened the pull request:
add component lines, replaced variables on the page OP

HaidaienkoK committed:
add component lines

biggujo commented on src/partials/lines.html:
```diff
@@ -0,0 +1,10 @@
+<section class="lines">
```
Це оформлення, тому це не повноцінна секція з контентом, а лише div
```suggestion
<div class="lines">
```

AlbinaKovalchuk replied:
++

biggujo commented on src/partials/lines.html:
```diff
@@ -0,0 +1,10 @@
+        <div class="line-three"></div>
+        <div class="line-four"></div>
+    </div>
+</section>
```
```suggestion
</div>
```

biggujo commented on src/sass/layout/_lines.scss:
```diff
@@ -0,0 +1,83 @@
+  margin-bottom: -5px;
+}
+
+.line_first,
```
Будь ласка, усі класи запишіть у форматі kebab-case: .line-first, .line-second тощо. Саме через мінус "-"

biggujo commented on src/sass/pages/_take-order.scss:
```diff
@@ -82,100 +49,15 @@ body {
-/* Конец линиям */
 
 @media screen and (min-width: 1280px) {
   .to__product_info {
```
Будь ласка, усі класи назвіть у форматі kebab-case: .to-product-info, .to-name тощо

HaidaienkoK replied:
Меню так працює по проекту...

biggujo reviewed (changes requested):
Добра робота. Прошу виконати такі рекомендації:

- додати margin для ліній: https://imgur.com/a/wjDGegH
- зробити випадне меню абсолютним: https://imgur.com/a/u7mP4RX
- takeOrder.html перейменувати на take-order.html
- виконати усі коментарі, що були дані до коду

HaidaienkoK committed:
bugs fixed, classes renamed

HaidaienkoK committed:
add button script, fixebag

HaidaienkoK committed:
redid the inputs, conteiner in to.scss

HaidaienkoK committed:
registered paths to scripts

HaidaienkoK committed:
fix paths to scripts

AlbinaKovalchuk committed:
Merge branch 'main' into feature/lines

AlbinaKovalchuk commented on src/sass/layout/_main.scss:
```diff
@@ -5,4 +5,6 @@
 @import 'hero';
 @import 'banner'; // Банер на головній сторінці
+@import 'lines';
 @import 'buttons'
```
треба додати крапку з комою  після @import 'buttons'

HaidaienkoK committed:
removed class in js files modal windows

HaidaienkoK committed:
Merge branch 'feature/lines' of https://github.com/one-lov-e/stalk into feature/lines

HaidaienkoK committed:
fix error name var

HaidaienkoK committed:
media 480px

HaidaienkoK committed:
mob menu

HaidaienkoK committed:
header and mob menu

Сделал header and mob menu. Не получилась наложить тень на моб меню....? Пришлось заново создать иконку закрытия окна.

biggujo committed:
added .is-hidden general class

biggujo committed:
final fix header + mobile menu

biggujo committed:
merge from main

biggujo committed:
deleted odd script for mobile menu

biggujo committed:
take-order.html fix

biggujo committed:
fix

biggujo reviewed (approved):
Добре
